CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/Popup` content padding and margin with `closeButton`
 - `agate/Popup` `bottom`, `fullscreen`, `left`, and `right` values to `position` prop
 - `agate/Popup` `duration` and `type` props
+- `agate/Switch` prop `disabled` to disable switch and become non-interactive
+- `agate/Switch` prop `selected` to set whether the control is in the 'on' or 'off' state
```

Comment:
Please remove the changelog since `agate/Switch` is a private component.

Reply:
Thanks. Good to know. I will share with the others as well

Reply:
Great! 👍